### DISCUSSION TITLE
Nvidium

### DISCUSSION THREAD

**Discussion by kakashi1464** [Sinytra/ConnectorExtras#14] [Ideas]
Nvidium support?

**Comment by Su5eD** [comment]
Please provide some details. I haven't tested nvidium on connector. Does it not work for you? What are the exact issues?

**Reply by kakashi1464** [reply]
Minecraft crashes when entering the world
[latest.log](https://github.com/Sinytra/ConnectorExtras/files/13296467/latest.log)
[crash-2023-11-08_18.06.15-client.txt](https://github.com/Sinytra/ConnectorExtras/files/13296536/crash-2023-11-08_18.06.15-client.txt)
[debug.log](https://github.com/Sinytra/ConnectorExtras/files/13296499/debug.log)

**Reply by kakashi1464** [reply]
Ok

**Comment by kakashi1464** [comment]
It seems nvidium is compatible with embeddium latest beta via Synitra Connector

**Reply by Saint4desu** [reply]
I can't seem to get it to work I changed the global alias but the version is still wrong when boot it up. how did you get it to work?

**Reply by SCPRedMage** [reply]
A bit late to the party, but you need to do two things, both done with files in your modpack's `config` folder.

First, you need to make sure Connector knows that Embeddium _is_ Sodium. You do this by adding an alias to `connector.json`; here is a sample version you can use, just replace the entire contents of your `connector.json` with this:
```
{
  "version": 1,
  "hiddenMods": [],
  "globalModAliases": {
    "cloth_config": "cloth-config2",
    "playeranimator": "player-animator",
    "embeddium": ["sodium"]
  }
}
```

Second, you need to remove the specific Sodium version number requirements Nvidium is looking for, because they won't match the Embeddium version numbers. You do this using a file called `fabric_loader_dependencies.json`; you likely won't have one, so create one and put this in it:
```
{
   "version": 1,
   "overrides":{
      "nvidium":{
         "-depends":{
            "sodium": "IGNORED"
         },
         "+depends":{
            "sodium": "*"
         }
      }
   }
}
```

With that added, it should work fine. I just used this today with Embeddium 0.3.20 and Nvidium 0.2.6-beta.
![image](https://github.com/Sinytra/ConnectorExtras/assets/17621208/1426360f-5517-4a32-a055-59824b025a96)
